### PR TITLE
PARQUET-297: Tests for PR 213 (Version generator)

### PR DESCRIPTION
**Pull request by isnotinvain**
Adds tests for #213 

How's this look @rdblue @kostya-sh ?

### CODE REVIEW

**Comment by rdblue** [comment]
+1

**Comment by isnotinvain** [comment]
Thanks!

**Comment by benpence** [comment]
+1

File: parquet-common/src/test/java/org/apache/parquet/VersionTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+ * being an empty string or something along those lines.
+ */
+public class VersionTest {
+
+  private void assertVersionValid(String v) {
+    try {
+      org.semver.Version.parse(v);
+    } catch (RuntimeException e) {
+      throw new RuntimeException(v + " is not a valid semver!" , e);
+    }
```
Comment (kostya-sh) [inline parquet-common/src/test/java/org/apache/parquet/VersionTest.java]: maybe use Assert.fail here? up to you

**Comment by kostya-sh** [comment]
+1

File: parquet-common/src/test/java/org/apache/parquet/VersionTest.java (hunk 1 above)
Comment (isnotinvain) [inline parquet-common/src/test/java/org/apache/parquet/VersionTest.java]: I wanted to originally, but fail() doesn't let you chain another exception, like `fail("message", e)` and I didn't want to drop the original exception from the logs. But I wish it did

**Pull request closed without merging**